src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -70,6 +70,11 @@ std::string getEntities
   string    idPattern  = ciP->uriParam["idPattern"];
   string    q          = ciP->uriParam["q"];
 
+  if (ciP->uriParam["options"] == "count")
```

Comment:
Found it :-)

uriParamOptionsParse() in ConnectionInfo.cpp is called by uriArgumentGet() in rest/rest.cpp.
There, all options in the URI param 'options' are put in the vector ciP->uriParamOptions.

So, I would propose to use

    if (ciP->uriParamOptions["count"] == true)

wherever needed, and probably remove these four lines.
[ It is a little ugly also to add parameters to ciP->uriParam ... it's like lying ... ]